deployments/examples/placements/README.md

```diff
@@ -20,13 +20,22 @@
 App placements rules in Yunikorn contains `Provided Rule`, `User Name Rule`, `Fixed Rule`, `Tag Rule`.
 Every placement example includes a example yaml file and a config yaml file.
 The sleep pod is described in the example file and the partition is provided in the config yaml file.
-Before deploying the pods, the configuration field in yunikorn-release/helm/yunikorn/value.yaml should be replaced by the configuration field in the config.yaml.
 
 * [Provided Rule](./provided)
 * [User Name Rule](./username)
 * [Fixed Rule](./fixed)
 * [Tag Rule](./tag)
 
+## How to adopt the queues in `config.yaml`
```

Comment:
We shouldn't just wipe out the default here. Instead, show the full configuration using the placement rules in `yunikornDefaults`, but also point out that the configuration can also be uploaded to the `yunikorn-configs` ConfigMap where it will take precedence. Editing `values.yaml` should be discouraged.

Reply:
This is the effective default, edit as necessary for your example here:
```
yunikornDefaults:
  queues.yaml: |
    partitions:
      - name: default
        placementrules:
          - name: tag
            value: namespace
            create: true
        queues:
          - name: root
            submitacl: '*'    
```

Reply:
Ok, will update this week.